game/models.py

```diff
@@ -179,6 +142,21 @@ def save(self, *args, **kwargs):
             self.created_at = timezone.now()
         return super(Task, self).save(*args, **kwargs)
 
+class DeployedTaskImage(models.Model):
+    task = models.ForeignKey(Task, related_name='deployed_images')
+    uploaded_image = models.ForeignKey('author.UploadedTaskImage', related_name='deployed_images')
+    suffix = models.CharField(max_length=1024)
+    host = models.CharField(null=False, blank=False, max_length=1024)
+    port = models.CharField(null=False, blank=False, max_length=5)
+
+    def __str__(self):
+        return self.uploaded_image.original_name.replace(".", "_") + "_" + self.suffix
```

Comment:
Suffix is tcp22 in this example:
{{task_name_tar_tcp22.port}}. We can rename it to sport.